viewer/templates/viewer/help.html

```diff
@@ -73,17 +80,15 @@ <h2>Components</h2>
 <div class="block">
   <h2>Links</h2>
   <p>
-    Find all instances of a string in the URLs of links listed on a page,
+    Find all instances of a string in the URLs of links listed on a cf.gov page,
     excluding the header and footer.
   </p>
   <p class="u-bold_text">Examples</p>
 
   <ul class="m-list">
     <li class="m-list_item">
       You're looking for all instances of external links to the FTC.
-      <a href="{% url 'index' %}?search_type=links&q=www.ftc.gov">
-        Search "www.ftc.gov"
-      </a>
+      <a href="{% url 'index' %}?search_type=links&q=FTC"> Search FTC </a>
```

Comment:
The design doesn't have underlines on these links. Is that intentional?

Reply:
No, Sonna said to keep the underlines